cosmotheka/mappers/mapper_ACT_compsept.py

```diff
@@ -12,14 +12,22 @@ class MapperACTCompSept(MapperACTBase):
     """
     def __init__(self, config):
         self._get_ACT_defaults(config)
+        if self.lmax > 3 * self.nside:
+            # WARNING
+            print("WARNING:              you selected lmax > 3*nside, "
+                  "setting lmax to 3 * nside.")
+            self.lmax = 3 * self.nside
 
     def _get_signal_map(self):
         # The 'Weights' FITS file contains the 2D Fourier space
         # weight for each pixel corresponding to the detector array.
         signal_map = enmap.read_map(self.file_map)
-        signal_map = reproject.healpix_from_enmap(signal_map,
-                                                  lmax=self.lmax,
-                                                  nside=self.nside)
+        signal_map = reproject.map2healpix(
+            signal_map,
+            nside=self.nside,
+            lmax=self.lmax,
+            niter=3
```

Comment:
Why `niter=3`? By default we use `niter=0` for the spherical harmonic transform. There were some effects with niter=3
https://github.com/Cosmotheka/Cosmotheka/blob/d66b83d28251a1323010399b44cfe76a85a57e35/input/master.yml#L83C1-L83C16